sdk/nodejs/runtime/closure/createClosure.ts

```diff
@@ -311,7 +311,12 @@ export async function createFunctionInfoAsync(
  */
 function createFunctionInfo(
         func: Function, context: Context,
-        serialize: (o: any) => boolean): FunctionInfo {
+        serialize: (o: any) => boolean, logInfo?: boolean): FunctionInfo {
+
+    // logInfo = logInfo || func.name === "addHandler";
+    if (logInfo) {
+        console.log("Serializing " + func.name);
+    }
```

Comment:
this is effectively debug-only code.  I end up adding this in a bunch, and i wouldn't mind just leaving this in.  Note that it will not do anything by default since logInfo=undefined.  But this allows for a pretty easy way to opt into turning on printf-style logging when helpful.
We could also consider some sort of logging library, but i didn't want to go through the pain of doing that.